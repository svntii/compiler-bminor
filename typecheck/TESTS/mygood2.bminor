a: auto = 10;
a: array [4] array [4] integer = {{1,2,3,4,5},{1,2,3,4,5},{1,2,3,4,5},{1,2,3,4,5},{1,2,3,4,5}};

//complex array
