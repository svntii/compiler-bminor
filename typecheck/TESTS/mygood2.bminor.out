	Scan Successful
	Parse Successful
	Reformatted Code
a: auto = 10;
a: array[] array[] integer = { { 1, 2, 3, 4, 5 }, { 1, 2, 3, 4, 5 }, { 1, 2, 3, 4, 5 }, { 1, 2, 3, 4, 5 }, { 1, 2, 3, 4, 5 } };

*****RESOLVING*******

	Code Resolved

*****TYPECHECKING*******

	Code Typecheck
